infinex-patron/patron-purchase-app/mainnets/infinex-patron.arbitrum.toml

```diff
@@ -0,0 +1,68 @@
+name = "infinex-patron"
+preset = "patron-purchase"
+version = "1.0.0"
+description = "Smart Contracts used for Infinex Patron Purchase App"
+
+# pnpm cannon build infinex-patron/patron-purchase-app/mainnets/infinex-patron.arbitrum.toml --chain-id 42161 --dry-run
+
+[var.PATRON]
+depends = []
+PATRON_DEPLOYER = "0xCD8AeD7674B10e1b6c8281B91259CEDE1BFE4a19"
+AUTHORIZED_SIGNER = "0x4f61e5795709C7A7E950Ea48c1cF5035F596d8A0"
+# AUTHORIZED MULTISIGS
+MULTISIG_0 = "0x4B190EC7A62EcE54c4DbF24F85D4fceF66c1446B"
+# AUTHORIZED TOKENS
+USDC_TOKEN = "0xaf88d065e77c8cC2239327C5EDb3A432268e5831" 
+USDT_TOKEN = "0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9"  
+USDE_TOKEN = "0x5d3a1Ff2b6BAb83b63cd9AD0787074081a52ef34"
+USDC_E_TOKEN = "0xff970a61a04b1ca14834a43f5de4533ebddb5cc8"
+DAI_TOKEN = "0xda10009cbd5d07dd0cecc66161fc93d7c9000da1"
+ARB_TOKEN = "0x912CE59144191C1204E64559FE8253a0e49E6548"
+ETHER = "0x0000000000000000000000000000000000000000"
+
+[clone.infinexPatronPurchase]
+depends = ['var.PATRON']
+options.PATRON_DEPLOYER = "<%= settings.PATRON_DEPLOYER %>"
+options.PATRON_PURCHASE_SALT = "PATRON_PURCHASE_SALT_1_0_13"
```

Comment:
is this the intended salt? I can see that the staging deployments used salts with similar format. It would have been better to append "staging" to the salts used for staging, so that there's no conflict with the prod salt format. In this case it shouldn't matter since the prod deployer will be a different address, but it's good practice to use separate formats so you don't accidentally have a conflict.